mandatory/1-fix-functions.js

```diff
@@ -10,20 +10,19 @@
 
 */
 
-function mood() {
-  let isHappy = true;
+function mood(isHappy) {
 
-  if (isHappy) {
+  if (isHappy === true) {
     return "I am happy";
   } else {
     return "I am not happy";
   }
 }
 
 function greaterThan10(num) {
-  let isBigEnough;
+  let isBigEnough = 10;
 
-  if (isBigEnough) {
+  if (num > isBigEnough) {
     return "num is greater than 10";
   } else {
     return "num is not big enough";
```

Comment:
Comment: I cannot find this mandatory among your answers here. 
 1. the user should be 18 or older
    2. the user must be logged in
*/
function isAcceptableUser(userAge, isLoggedIn) {}
function isAcceptableUser(userAge, isLoggedIn) {
  if (userAge >=18 && isLoggedIn) {
return true
  } else {
return false
  }
}